LibTrixi.jl/src/api_c.jl

```diff
@@ -328,32 +373,60 @@ trixi_nvariables_cfptr() = @cfunction(trixi_nvariables, Cint, (Cint,))
 
 
 """
-    trixi_load_cell_averages(data::Ptr{Cdouble}, simstate_handle::Cint)::Cvoid
+    trixi_load_cell_averages(data::Ptr{Cdouble}, index::Cint, simstate_handle::Cint)::Cvoid
 
 Return cell averaged solution state.
```

Comment:
I hadn't noticed before, but I think "Return" is the wrong word here, since the function does not return anything...